win/pkgs/tensorrt.ps1

```diff
@@ -42,7 +42,7 @@ Move-Item -Force "${trt_name}.extracting.d/TensorRT-*" "tensorrt"
 rm -Force -Recurse "${trt_name}.extracting.d"
 
 rm -Force -Recurse -ErrorAction SilentlyContinue -WarningAction SilentlyContinue "${Env:ProgramFiles}/tensorrt"
-Move-Item -Force "tensorrt" "${Env:ProgramFiles}/tensorrt"
+Copy-Item -Recurse -Force "tensorrt" "${Env:ProgramFiles}/tensorrt"
```

Comment:
What's the error?
Why will move fail?
If someone still opened the file, copy can cause data corruption as well.

Reply:
I couldn't find the original error log. It was about the system couldn't find some files although the package has been unpacked to the desired location. Using `Copy-Item` fixed the issue.

I'll revert this change and see if the issue remains.

Reply:
Reverted, not able to repro the issue any more.